Foundation/include/Poco/Any.h

```diff
@@ -248,12 +272,16 @@ class Any
 			construct(other);
 	}
 
-	~Any()
-		/// Destructor. If Any is locally held, calls ValueHolder destructor;
-		/// otherwise, deletes the placeholder from the heap.
+	Any(Any&& other)
+		/// Move constructor, works with both empty and initialized Any values.
 	{
+		construct(other);
 	}
 
+	~Any() = default;
+		/// Destructor. If Any is locally held, calls ValueHolder destructor;
+		/// otherwise, deletes the placeholder from the heap.
```

Comment:
This comment requires searching through implementation to understand what is really going on. Maybe something like this:
```c++
/// Destructor.
/// Small Object Optimization mode behavior:
/// Invokes the Placeholder destructor, which calls the
/// ValueHolder destructor explicitly (locally held Any), or
/// deletes the ValueHolder heap memory (heap-allocated Any).
```